testing/conformance/benches/bench_conformance_overhead.rs

```diff
@@ -0,0 +1,117 @@
+extern crate criterion;
+use std::env::var;
+use std::path::{Path, PathBuf};
+use std::time::Duration;
+
+use conformance_tests::test_utils::*;
+use conformance_tests::vector::ApplyMessage;
+use criterion::*;
+use fvm::machine::BURNT_FUNDS_ACTOR_ADDR;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::encoding::{Cbor, RawBytes};
+use fvm_shared::message::Message;
+use walkdir::WalkDir;
+
+mod bench_utils;
+use crate::bench_utils::{bench_vector_file, CheckStrength};
+
+fn bench_no_messages(
+    group: &mut BenchmarkGroup<measurement::WallTime>,
+    path_to_setup: PathBuf,
+) -> anyhow::Result<()> {
+    // compute measurement overhead by benching running a single empty vector of zero messages
+    match &bench_vector_file(
+        group,
+        path_to_setup,
+        Some(vec![]),
+        true,
+        Some("bench_no_messages".parse().unwrap()),
+        CheckStrength::OnlyCheckSuccess,
+    )?[0]
+    {
+        VariantResult::Ok { .. } => Ok(()),
+        VariantResult::Skipped { reason, id } => Err(anyhow::anyhow!(
+            "no messages test {} skipped due to {}",
+            id,
+            reason
+        )),
+        VariantResult::Failed { reason, id } => Err(anyhow::anyhow!(
+            "no messages test {} failed due to {}",
+            id,
+            reason
+        )),
+    }
+}
+
+fn bench_noops(
```

Comment:
Rust docs. Let's rename to `bench_simple_state_access`. This is technically not a noop.